docs/commands/zq.md

```diff
@@ -172,18 +174,32 @@ typically omit quotes around field names.
 
 ## Output Formats
 
+`zq` currently supports the following output formats:
+
+|  Option   | Specification                            |
+|-----------|------------------------------------------|
+| `arrows`  | [Arrow IPC Stream Format](https://arrow.apache.org/docs/format/Columnar.html#ipc-streaming-format) |
+| `csv`     | [CSV RFC 4180](https://www.rfc-editor.org/rfc/rfc4180.html) |
+| `json`    | [JSON RFC 8259](https://www.rfc-editor.org/rfc/rfc8259.html) |
+| `parquet` | [Apache Parquet](https://github.com/apache/parquet-format) |
+| `table`   | (described [below](#simplified-text-outputs)) |
+| `text`    | (described [below](#simplified-text-outputs)) |
+| `tsv`     | [TSV - Tab-Separated Values](https://en.wikipedia.org/wiki/Tab-separated_values) |
+| `vng`     | [VNG - Binary Columnar Format](../formats/vng.md) |
+| `zeek`    | [Zeek Logs](https://docs.zeek.org/en/master/logs/index.html) |
+| `zjson`   | [ZJSON - Zed over JSON](../formats/zjson.md) |
+| `zng`     | [ZNG - Binary Row Format](../formats/zson.md) |
+| `zson`    | [ZSON - Human-readable Format](../formats/zson.md) |
```

Comment:
I imagine some folks may not love the repetition between this and the "inputs" table. There's a couple reasons I bit the bullet and added this.

1. The community user that kicked off this effort expressed a sense that they expected to see the table. The text that was there before said "The supported output formats include all of the input formats along with text and table formats", so I'd have hoped that was enough coverage. But I know that users can often be in a rush when consulting docs.

2. This is sort of a variation on the previous point, but given the section header hyperlinks for "Input Formats" and "Output Formats" in the right-hand side of the screen when viewing the rendered docs site, it is indeed a handy convenience to see the quick splash table with one click rather than having to do a "delta" exercise after clicking to another section.

3. Strictly speaking, we can't claim that the supported output formats include all of the input formats, since `line` is not an output format (incidentally, that's been requested by at least one user via #5042).